folksonomy/api.py

```diff
@@ -291,6 +291,14 @@ async def product_stats(response: Response,
     # )
     # out2 = await cur.fetchone()
     # import pdb;pdb.set_trace()
+
+    if not out or not out[0]:
+        return JSONResponse(
+            status_code=200,
+            content={"message": "No products found matching the given filters."},
```

Comment:
Why put a message here? When the list is not empty, the answer is a list.
When the list is empty, it should return an empty list IMHO. Code will be easier to write for API users.
```suggestion
            content=[],
```